engine/client/gl_local.h

```diff
@@ -349,6 +349,72 @@ void R_LightForPoint( const vec3_t point, color24 *ambientLight, qboolean invLig
 int R_CountSurfaceDlights( msurface_t *surf );
 int R_CountDlights( void );
 
+
+/// <STROBE> // Move to strobe.h ?
```

Comment:
Yeah, it would be a good idea. 

Because all source code is categorized, you may want to move code to "r_strobe.c" and "r_strobe.h" files.